### PR TITLE
add_alias

### PR DESCRIPTION
**Pull request by shantanu-deriv**
https://redmine.deriv.cloud/issues/26400#correct_POD_and_add_alias

### CODE REVIEW

File: lib/Date/Utility.pm [hunk 1]
```diff
@@ -454,7 +454,7 @@ sub new {
 
 }
 
-=head2 _parse_datetime_parm
+=head2 _parse_datetime_param
```
Comment (shantanu-deriv) [inline lib/Date/Utility.pm:457]: this is just a typo fix

**Comment by shantanu-deriv** [comment]
How to add [changes](https://github.com/binary-com/perl-Date-Utility/blob/master/Changes)?

File: lib/Date/Utility.pm [hunk 2]
```diff
@@ -1167,15 +1167,17 @@ e.g.
 
 =cut
 
-sub _plus_years {
+sub plus_years {
     my ($self, $years) = @_;
```
Comment (sasikala-binary) [inline lib/Date/Utility.pm:1179]: Do we need to change https://github.com/binary-com/perl-Date-Utility/pull/33/files#diff-f5a42ba81e6bad2f023037d764e416a3a8344f43734f9671dd3afd2e2b7c5488R1165 
to print Date::Utility->new('2000-02-29')->plus_years(1)->date_yyyymmdd; to consistence with https://github.com/binary-com/perl-Date-Utility/pull/33/files#diff-f5a42ba81e6bad2f023037d764e416a3a8344f43734f9671dd3afd2e2b7c5488R1202 and https://github.com/binary-com/perl-Date-Utility/pull/33/files#diff-f5a42ba81e6bad2f023037d764e416a3a8344f43734f9671dd3afd2e2b7c5488L1183

File: lib/Date/Utility.pm [hunk 3]
```diff
@@ -1157,7 +1157,7 @@ sub today {
     return $today_obj;
 }
 
-=head2 _plus_years
+=head2 plus_years
```
Comment (sasikala-binary) [inline lib/Date/Utility.pm:1160]: could we update the pod's based on style guide https://wikijs.deriv.cloud/en/Backend/guides/perl/perl-style-guide#subroutine-comments

File: lib/Date/Utility.pm [hunk 4]
```diff
@@ -1227,25 +1233,29 @@ e.g.
...
-sub _create_trimmed_date {
+sub create_trimmed_date {
     my ($self, $year, $month, $day) = @_;
     my $max_day = __PACKAGE__->new(sprintf("%04d-%02d-01", $year, $month))->days_in_month;
     $day = $day < $max_day ? $day : $max_day;
     my $date_string = sprintf("%04d-%02d-%02d %02d:%02d:%02d", $year, $month, $day, $self->hour, $self->minute, $self->second);
     return __PACKAGE__->new($date_string);
 }
 
+*_create_trimmed_date = \&create_trimmed_date;
```
Comment (chylli-deriv) [inline lib/Date/Utility.pm:1301]: Do we have several tests to test those `_*` methods ? If not, should we add some ?

File: lib/Date/Utility.pm (hunk 3 above)
Comment (shantanu-deriv) [reply lib/Date/Utility.pm:1160]: added pod

File: lib/Date/Utility.pm (hunk 2 above)
Comment (shantanu-deriv) [reply lib/Date/Utility.pm:1179]: fixed usage example

File: lib/Date/Utility.pm (hunk 4 above)
Comment (shantanu-deriv) [reply lib/Date/Utility.pm:1301]: added tests

**Pull request merged**